### PR TITLE
feat(controller): provide way to go back to join code

### PR DESCRIPTION
**Pull request by virtuacoplenny**
Screen Shot 2019-06-20 at 9 35 51 AM

### CODE REVIEW

File: spot-controller/App.js [hunk 1]
```diff
@@ -127,7 +132,20 @@ export default class App extends React.Component {
...
         return (
-            <SettingsMenu onClearRemoteUrl = { this._onClearRemoteUrl } />
+            <SettingsMenu
+                onClearRemoteUrl = { this._onClearRemoteUrl }
+                onResetApp = { this._onResetApp } />
         );
     }
+
+    /**
+     * Ah shit.
```
Comment (paweldomas) [inline spot-controller/App.js]: :)

Comment (paweldomas) [reply spot-controller/App.js]: lol

Comment (virtuacoplenny) [reply spot-controller/App.js]: This is what my code looks like without its makeup.

**Comment by virtuacoplenny** [comment]
> LGTM why is this webview key needs to be changed ?

Looks like I lost my comment about it, which I will add back. I couldn't find a decent way to make the webview go back to the initial URL. So instead I force a re-render by changing the key.

**Pull request merged**